src/resources/views/columns/image.blade.php

```diff
@@ -0,0 +1,18 @@
+<td>
+  @if( !empty($entry->{$column['src']}) )
+    <a
+      href="{{ $entry->{$column['src']} }}"
+      target="_blank"
+    >
+      <img
+        src="{{ $entry->{$column['src']} }}"
+        style="
+          height: {{ isset($column['height']) ? $column['height'] : "25px" }};
+          width: {{ isset($column['width']) ? $column['width'] : "auto" }};
```

Comment:
See above - but this time why 'auto'?

Reply:
Apart from the explanation that was missing for the choice of `25px` for the height, I don't see what's wrong by setting a fixed value as height and `auto` as width ? Do you know a better way ?

Btw `height` could be replaced by  `max-height` to prevent oversizing the image if it's height is smaller.

Reply:
Possibly max-height actually :)

Reply:
I personally like ```width:auto```. I'm all for keeping the aspect ratio.